api_tests/base/test_sloan_study.py

```diff
@@ -87,12 +84,9 @@ def test_sloan_study_variable(self, app, user, preprint):
         assert f' dwf_{SLOAN_DATA_DISPLAY}=True; Domain=localhost; Path=/; samesite=None; Secure' in cookies
         assert f' dwf_{SLOAN_PREREG_DISPLAY}=True; Domain=localhost; Path=/; samesite=None; Secure' in cookies
 
-    @override_settings(DEV_MODE=False)
     @pytest.mark.enable_quickfiles_creation
     @mock.patch('waffle.models.Decimal', inactive)
     def test_sloan_study_control(self, app, user, preprint):
-        Flag.objects.filter(name__in=SLOAN_FLAGS).update(percent=1)
```

Comment:
ug simply forgot to remove this Flag the same is true of [this one](https://github.com/CenterForOpenScience/osf.io/pull/9336/files#diff-35f5a3f49524bbaf7aee4d9e75149182L134)